tests/test_fs.md

```diff
@@ -31,6 +31,11 @@ let try_mkdir dir path =
   | () -> traceln "mkdir %S -> ok" path
   | exception ex -> traceln "mkdir %S -> %a" path Fmt.exn ex
 
+let try_read_dir dir path =
+  match Eio.Dir.read_dir dir path with
+  | names -> traceln "read_dir [ %a ] -> ok" Fmt.(list ~sep:Fmt.comma string) names
+  | exception ex -> traceln "read_dir %a" Fmt.exn ex
```

Comment:
```suggestion
  | names -> traceln "read_dir %S -> %a" path Fmt.Dump.(list string) names
  | exception ex -> traceln "read_dir %S -> %a" path Fmt.exn ex
```
Just for consistency with the others. `mkdir` only prints `ok` because there's nothing else to print.